app/src/main/java/com/aptoide/uploader/account/AutoLoginManager.java

```diff
@@ -6,56 +6,113 @@
 import io.reactivex.Single;
 
 public class AutoLoginManager {
-  private Context context;
-  private AutoLoginPersistence persistence;
+  private static Context context;
+  private AutoLoginCredentials autoLoginCredentials = new AutoLoginCredentials();
+  private static AutoLoginManager single_instance;
 
-  public AutoLoginManager(Context context, AutoLoginPersistence persistence) {
+  public AutoLoginManager(Context context) {
     this.context = context;
-    this.persistence = persistence;
+  }
+
+  public static AutoLoginManager getInstance(Context context) {
+    if (single_instance == null) single_instance = new AutoLoginManager(context);
+    return single_instance;
+  }
+
+  public AutoLoginCredentials getAutoLoginCredentials() {
+    return autoLoginCredentials;
   }
 
   public Single<AutoLoginCredentials> getStoredUserCredentials() {
+    return Single.just(autoLoginCredentials);
+  }
+
+  public Single<AutoLoginCredentials> getFirstStoredUserCredentials() {
     try {
       String URL = "content://cm.aptoide.pt.StubProvider";
       Uri token_uri = Uri.parse(URL + "/token");
       Uri refresh_token_uri = Uri.parse(URL + "/refreshToken");
       Uri repo_uri = Uri.parse(URL + "/repo");
+      Uri email_uri = Uri.parse(URL + "/loginName");
+      Uri nickname_uri = Uri.parse(URL + "/loginNickname");
+      Uri avatar_uri = Uri.parse(URL + "/loginAvatar");
```

Comment:
I totally agree. I didnt changed it because I had to do it in vanilla repo as well and i though it could break something but to simplify, I wont change the variables names in vanilla and will only change the strings sent in the ContentProvider

Reply:
https://github.com/Aptoide/aptoide-client-v8/pull/1393#discussion_r481185603

Reply:
so, we shouldn't use the "_uri" part? already changed the nickname to name and the repo to store_name.. but the uri stayed

Reply:
In the future we change, _uri is not a valid naming pattern. Snake case in Java is only used for static constants https://en.wikipedia.org/wiki/Snake_case